test/execution_summary_test.py

```diff
@@ -1086,3 +1088,87 @@ class Bar(RunOnceTask):
         self.assertIn('Luigi Execution Summary', s)
         self.assertNotIn('00:00:00', s)
         self.assertNotIn('\n\n\n', s)
+
+class ExecutionSummaryWithTaskRetryTest(LuigiTestCase):
```

Comment:
Sorry for ugly test case...

I could not a smart test case with the existing one.
After struggling, I have found it can by using 

```
luigi.scheduler.Scheduler(prune_on_get_work=True, retry_delay=1)
luigi.worker.Worker(scheduler=self.scheduler, keep_alive=True)
```

and flag file which controls the target task failure and success.
By them, the task once fails, and the scheduler re-assign the task quickly and the task suceeds in second try. `os.sleep()` is just a safety for waiting task re-assign.

I will keep trying to merge them into the existing class and removing ugly trick with file system and sleep.
If there is a good trick, please let me know.